app/graphql/types/base_argument.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Types
+  class BaseArgument < GraphQL::Schema::Argument; end
+end
```

Comment:
Can you explain the purpose of this and the following classes / why they need to be broken out? I would personally rather tap into `GraphQL::Schema::Enum` directly unless that type (and the following below) need to be extended significantly. Until then it seems a bit premature.